src/config.ts

```diff
@@ -132,13 +134,29 @@ export const CLIENT_ERROR_REDIRECTION_URL =
 export const CLIENT_REDIRECTION_URL =
   process.env.CLIENT_REDIRECTION_URL || "/login";
 
+const SPID_LEVEL_WHITELIST = fromNullable(process.env.SPID_LEVEL_WHITELIST)
+  .map(_ => _.split(","))
+  .foldL(
+    // If SPID_LEVEL_WHITELIST is unset, all the spid levels are enabled
```

Comment:
I'd default to `["SpidL1", "SpidL2", "SpidL3"]` if NODE_ENV=dev and `["SpidL2", "SpidL3"]` otherwise

Reply:
Whit this change in dev mode you cannot test Spid level whitelisting. Could be a problem?

Reply:
Ok, it's only for the default value. I make the changes.

Reply:
Fixed in 468f737